### ISSUE TITLE
Port #963 (Cassandra widerows) to titan09

### ISSUE THREAD

**Issue by dalaro** [thinkaurelius/titan#1075]
The Hadoop inputformats were completely rewritten between titan05 and titan09.  Cassandra widerow support was implemented on titan05, but those changes no longer apply meaningfully on titan09.

   Related commits:
- 3c776d48e6d7d40359c2d7080409fd590d695728
- 3736b37212d3a9e2278e44e4b070ba02b32cceea

**Comment by dalaro** [comment]
Insofar as afffa0794213ce1094b66668d2d4aaafd534bc10 is actually correct, this just needs a test case.

**Comment by dalaro** [comment]
I realized just after my last comment that the necessary test already exists (5e5b6b4b4bc1cdcdb6078a4cf4fee89b324ec94a) and is still present.  However, titan09 had a couple of completely unrelated problems causing exceptions in titan-hadoop-2 integration test execution that I had to fix before I could successfully execute the widerows test.  Fixes to the unrelated IT problems: dcedc76f79da40cc1d3ad0a440500b41dbc59bfa and 218a1153bf031020fbf95bd56df9a6d9146ca030.

This shows terrible issue hygiene on my part.  I should have closed this in May before the intervening Hadoop integration test issues cropped up.  But now it's closed.
